### PR TITLE
Fix: console header error

### PR DESCRIPTION
**Pull request by hamyhya**

### CODE REVIEW

**Comment by txalkan** [comment]
the package-lock should not be included, right? pls merge the last version of dev into yours

File: package-lock.json
Comment (txalkan) [inline package-lock.json:1]: I meant to delete this file from the PR (not the codebase) - how to do this?

**Review by txalkan — CHANGES_REQUESTED** [review]
please remove the `package-lock` from this PR (unless there is a reason for it to be included?)

and ask for my review afterwards, thanks

File: package-lock.json
Comment (hamyhya) [reply package-lock.json:1]: @txalkan I have revert the package lock using same as from dev, now if you load the different here should be not much change

**Review by txalkan — APPROVED** [review]

**Pull request merged**